src/aleph/vm/orchestrator/supervisor.py

```diff
@@ -107,6 +107,10 @@ async def stop_all_vms(app: web.Application):
 def run():
     """Run the VM Supervisor."""
     settings.check()
+
+    engine = setup_engine()
+    create_tables(engine)
```

Comment:
Are there migrations to run ?

Reply:
I don't think so, also the `EXECUTIONS_DATABASE` parameter was not well set, and I already fixed it to use the correct path, so it will use a new dabase.

Reply:
But the renamed database above might be a problem

Reply:
I don't think so, because we aren't using it now.